### DISCUSSION TITLE
Note taking capabilities inside IPED?

### DISCUSSION THREAD

**Discussion by milcent-CVM** [sepinf-inc/IPED#1852] [General]
Is there a way of taking notes about a specific document/artifact inside IPED?

**Comment by lfcnassif** [comment]
Unfortunately not a specific feature for this, but you can create a bookmark with a single artifact and add a comment to it.

**Comment by milcent-CVM** [comment]
Thanks,
I saw this [other discussion](https://github.com/sepinf-inc/IPED/discussions/1839) with the script that adds the transcription to the metadata, but this happens in processing time, right?
The note-taking capability I imagined would have to be post-processing, in analysis time, kind of previous to generating a report or just to have it there for remembering important things about the artifacts that are too long to fit in a bookmark.
Thanks anyway!

**Reply by lfcnassif** [reply]
> I saw this https://github.com/sepinf-inc/IPED/discussions/1839 with the script that adds the transcription to the metadata, but this happens in processing time, right?

Yes

**Reply by lfcnassif** [reply]
> The note-taking capability I imagined would have to be post-processing, in analysis time, kind of previous to generating a report or just to have it there for remembering important things about the artifacts that are too long to fit in a bookmark.

Maybe #452 would fit your needs. Unfortunately it is low priority right now, but if you have available time to help the project, contributions would be very welcome!
